lib/includes/nghttp2/nghttp2.h

```diff
@@ -4519,6 +4519,38 @@ nghttp2_hd_deflate_hd(nghttp2_hd_deflater *deflater, uint8_t *buf,
 /**
  * @function
  *
+ * Deflates the |nva|, which has the |nvlen| name/value pairs, into
+ * the |inlen| size of buf vector |bufsin|, each buf has length |buflen|.
+ *
```

Comment:
It would be nice to tell the reader that if one chunk is used up, next chunk is used.